std/string.d

```diff
@@ -2882,6 +2926,12 @@ if (isConvertibleToString!Range)
     return stripLeft!(StringTypeOf!Range)(str);
 }
 
+auto stripLeft(Range)(auto ref Range str, string chars)
```

Comment:
These two overloads also need to have the right order.  Here's what the order should be:
````
/** ... */
auto stripLeft(...) ... // original stripLeft

///
unittest { ... } // original test cases

/// ditto   <--- add this comment
auto stripLeft(Range)(auto ref Range str)
if (isConvertibleToString!Range) ...

///
unittest { ... } // more original test cases

/** ... */
auto stripLeft(...) ... // new stripLeft

///
unittest { ... } // new test cases

/// ditto   <--- add this comment
auto stripLeft(Range)(auto ref Range str, string chars)
if (isConvertibleToString!Range) ...

///
unittest { ... } // more new test cases (if there are any specific to the 2nd overload)
````